frontend/src/metabase/containers/AddToDashSelectDashModal/AddToDashSelectDashModal.unit.spec.tsx

```diff
@@ -343,9 +342,13 @@ describe("AddToDashSelectDashModal", () => {
         });
 
         describe('"Create a new dashboard" option', () => {
-          it('should render "Create a new dashboard" option when opening the root collection (public collection)', async () => {
-            await setup();
+          beforeEach(async () => {
+            await setup({
+              collections: COLLECTIONS,
```

Comment:
without passing `collections` it's already defaulted to `COLLECTIONS` but I want to make it explicit. So, it's more obvious what `collections` we're using in each test.

https://github.com/metabase/metabase/blob/7948e531affb55580e9f9911e6c3c00e8b5cb32a/frontend/src/metabase/containers/AddToDashSelectDashModal/AddToDashSelectDashModal.unit.spec.tsx#L120

Reply:
Should we remove the default value and require `collections` attribute then?

Also, we could reuse `COLLECTIONS` in assertions, e.g.

```js
for (const { name } of COLLECTIONS) {
  expect(
    await within(popover).getByRole("heading", { name }),
  ).toBeInTheDocument();
}
```